src/main/java/frc/robot/Constants.java

```diff
@@ -88,6 +88,12 @@ public static class ArmConstants {
    * Constants for the game piece manipulator
    */
   public static class GamePieceManipulatorConstants {
+    public static final AngularVelocity INTAKESPEED = RadiansPerSecond.of(5);
 
+    public static final AngularVelocity OUTTAKESPEED = RadiansPerSecond.of(-5);
+
+    public static final AngularVelocity SCORESPEED = RadiansPerSecond.of(5);
+
+    public static final int DEVICE_ID_MANIPULATORMOTOR = 5;
```

Comment:
There should be an _ between words in these names.
Constant names are SNAKE_CASE, meaning they're all upper case with underscores to separate words.